AdminWebsite/AdminWebsite.AcceptanceTests/appsettings.json

```diff
@@ -9,7 +9,8 @@
         "Authority": "https://login.microsoftonline.com/",
         "GraphApiBaseUri": "https://graph.microsoft.com/",
         "RedirectUri": " https://localhost:5400/login",
-        "PostLogoutRedirectUri": "https://localhost:5400/"
+        "PostLogoutRedirectUri": "https://localhost:5400/",
+        "ResourceId": "resourceID"
```

Comment:
don't think the `ResourceId` is needed here. It gets replaced by the client id in the frontend:

```
configService.getClientSettings().subscribe(clientSettings => {
            const resource = clientSettings.resource_id ? clientSettings.resource_id : `api://${clientSettings.client_id}`;
```

Reply:
You're right, it's not needed in the pipeline. It is needed if you want to run the admin web Acceptance Tests locally, else it throws an exception that the value is missing when when it tries to parse the file in the configuration hook. I figured I'd leave the change in so people dont need to add it in, in future.

Reply:
right ok. I already have the ResourceId in secrets, probably why it didn't fail for me. But yes, that sounds fine